release-notes-2.2.0.md

```diff
@@ -4,5 +4,7 @@
 ${version-number}
 
 #### New Features
+- **US922204**: Autoscaler is only available using Kubernetes
```

Comment:
Think this is a breaking change?

Reply:
Yeah I wasn't sure about this, will change to breaking change thanks.